### PR TITLE
Support multiple sdc files

### PR DESCRIPTION
**Pull request by alain-rs**

### CODE REVIEW

**Comment by alain-rs** [comment]
Prevent reset of Constraint object in between files.
Parse constraint files only once when added, and again when synthesis is ran

File: src/Compiler/Compiler.cpp [hunk 1]
```diff
@@ -700,6 +708,12 @@ bool Compiler::RegisterCommands(TclInterpreter* interp, bool batchMode) {
         compiler->ErrorMessage(out.str());
         return TCL_ERROR;
       }
+    } else {
```
Comment (KochynVolodymyr) [inline src/Compiler/Compiler.cpp:711]: this is not working. why it was changed?

Comment (alaindargelas) [reply src/Compiler/Compiler.cpp:711]: Constraint file was being sourced twice. One in this method, another time in the project method. 
Especially obvious when multiple constraint files are used.
What is not working exactly? It is working fine with all the tests I did with multiple constraint files.

Comment (KochynVolodymyr) [reply src/Compiler/Compiler.cpp:711]: This code is not reachable since compiler->m_tclCmdIntegration always true

**Comment by alaindargelas** [comment]
Then just remove this redundant code. It works with the other code path since you read_sdc in the other function.

**Pull request merged**